tests/db_utils/tests.py

```diff
@@ -90,3 +105,80 @@ def test_load_backend_invalid_name(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg) as cm:
             load_backend("foo")
         self.assertEqual(str(cm.exception.__cause__), "No module named 'foo'")
+
+
+class AsyncConnectionTests(SimpleTestCase):
```

Comment:
`test_new_connection_threading` and `test_new_connection` fails on postgres with connection pools:
`AttributeError: 'coroutine' object has no attribute 'set_autocommit'`

Reply:
fixed in 9ddddab18b7fe155bfd7fb99f897fbaf2ecce6a6